Serializer/BaseSerializerHandler.php

```diff
@@ -27,6 +27,11 @@
     protected $manager;
 
     /**
+     * @var array
+     */
+    private static $formats = array('json', 'xml', 'yml');
```

Comment:
It has to be hardcoded with the injection option. Otherwise it will be a BC break. I can set this default array in constructor also.